app/main/views/marketplace.py

```diff
@@ -123,50 +132,146 @@ def get_brief_by_id(framework_framework, brief_id):
-        prev_link_args.setlist("page", api_prev_link_args.get("page") or ())
-
-    api_next_link_args = parse_link(links, "next")
-    next_link_args = None
-    if api_next_link_args:
-        next_link_args = request.args.copy()
-        next_link_args.setlist("page", api_next_link_args.get("page") or ())
-
-    return render_template('search/briefs.html',
-                           framework=frameworks[-1],
-                           form=form,
-                           filters=form.get_filters(),
-                           filters_applied=form.filters_applied(),
-                           briefs=briefs,
-                           lot_names=tuple(label for id_, label in form.lot.choices),
-                           prev_link_args=prev_link_args,
-                           next_link_args=next_link_args,
-                           briefs_count=api_result.get("meta", {}).get("total", None),
-                           )
+    pagination_config = pagination(
+        search_results_obj.total,
+        current_app.config["DM_SEARCH_PAGE_SIZE"],
+        get_page_from_request(request)
+    )
+
+    search_summary = SearchSummary(
+        search_api_response['meta']['total'],
+        clean_request_query_params.copy(),
+        filters.values(),
+        lots_by_slug
+    )
+
+    category_filter_group = filters.pop('categories') if 'categories' in filters else None
+    lots = [lot for lot in framework['lots'] if lot['allowsBrief']]
+
+    view_name = 'list_opportunities'
+    selected_category_tree_filters = build_lots_and_categories_link_tree(
+        framework,
+        lots,
+        category_filter_group,
+        request,
+        updated_request_args if updated_request_args else clean_request_query_params,
+        content_manifest,
+        doc_type,
+        index,
+        Href(url_for('.{}'.format(view_name), framework_family=framework['framework'])),
+    )
+
+    filter_form_hidden_fields_by_name = {f['name']: f for f in selected_category_tree_filters[1:]}
+    current_lot = lots_by_slug.get(current_lot_slug)
+
+    set_filter_states(filters.values(), request)
+
+    for filter_groups in filters.values():
+        for filter_instance in filter_groups['filters']:
+            if 'label' in filter_instance:
+                filter_instance['label'] = capitalize_first(filter_instance['label'])
+
+    clear_filters_url = get_request_url_without_any_filters(
+        request, filters, view_name, framework_family=framework_family
+    )
+    search_query = query_args_for_pagination(clean_request_query_params)
+
+    template_args = dict(
+        briefs=search_results_obj.search_results,
+        category_tree_root=selected_category_tree_filters[0],
+        clear_filters_url=clear_filters_url,
+        current_lot=current_lot,
+        doc_type=doc_type,
+        filters=filters.values(),
+        filter_form_hidden_fields=filter_form_hidden_fields_by_name.values(),
+        framework=frameworks[-1],
```

Comment:
`-1` ?

Reply:
In fact we already have a variable `framework` (set to the latest framework in the family) so probably we should be consistent and use that, rather than just the last one in the list?